.rustfmt.toml

```diff
@@ -0,0 +1 @@
+use_small_heuristics = "Max"
```

Comment:
This made the code more compact, the sizages were blown out over multiple lines otherwise.